README.rst

```diff
@@ -491,4 +491,6 @@ LICENSE <https://github.com/ni/nimi-python/blob/master/LICENSE>`_).
 Other incorporated projects may be licensed under different licenses. All
 licenses allow for non-commercial and commercial use.
 
+Usage of GrpcSessionOptions on session initialization are subject to the NI General Purpose EULA (`see NILICENSE <https://github.com/ni/nimi-python/blob/master/NILICENSE>`_).
```

Comment:
Is a url better or a relative path?

Reply:
I just followed the same pattern for the previous comment about the MIT license.